_posts/2022-09-09-update-covid-policy.md

```diff
@@ -0,0 +1,43 @@
+---
+author: Drew Winstel
+category: General
+date: 2022-09-09 12:00:00-07:00
+layout: post
+title: Updates to our COVID-19 Policy
+---
+
+In light of the CDC's [approval of the new bivalent boosters](https://www.cdc.gov/media/releases/2022/s0901-covid-19-booster.html), we have updated our COVID-19 policy.
+
+# COVID-19 Policy updates
+
+## Booster requirements
+
+Given that there are only 6 weeks between the launch of the bivalent boosters and the start of the conference, we will be _strongly_ encouraging but not requiring the new booster.
+
+Instead, here are the requirements:
+
+### If you have not had any boosters
+
+If it has been at least 5 months since your last dose of your initial vaccination schedule (as of October 16, 2022), you are required to get a booster shot if it is available in your country. Otherwise, you are not required to get a booster.
+
+### If you have had any boosters
+
+If you have had at least one booster shot previously (i.e. you were considered up to date under the old vaccination guidelines), you are not required to get an additional booster.
+
+## Masking policies
+
+As N95 is an American standard, some attendees have inquired about whether we will allow attendees to bring their own masks certified by other governments. We will allow certified respirators certified as having met standards comparable to N95, such as FFP2 (European Union), P2 (Australia/New Zealand), DS (Japan), and KMOEL-2017-64 (South Korea, also called KF94).
+
+Cloth masks and reusable masks which are not certified are not allowed.
+
+We will provide N95 masks at registration, so you will be covered either way.
+
+## Proof of negative test
+
+We will require proof of a negative COVID-19 test before you are allowed to claim your badge.
+
+For PCR tests, the test must be taken no earlier than 3 days before the day of the event (e.g. for the tutorials on Sunday, October 16, your test can be no earlier than Thursday, October 13).
+
+For antigen and other such rapid tests that deliver results in under 24 hours, the test must be taken no earlier than the day before the event.
+
+If you are feeling even slightly unwell, please come to the registration desk to obtain a rapid test. In the event of a positive test while you are at DjangoCon, we will give you early access to the online platform so you are able to watch the rest of the talks from your hotel room.
```

Comment:
We should perhaps make it a little more clear that we either have enough tests for everyone or that we encourage folks to ger tested before and use their own tests. Right now, it could be a bit ambiguous.

Reply:
We'll be ordering a bunch.